app/page.tsx

```diff
@@ -25,6 +26,7 @@ const Home = () => {
       <ProjectsSection />
       <FAQ data={FAQ_DUMMY_DATA} />
       <Contact />
+      <Table />
```

Comment:
We shouldn't render this component in the main page.
We will only use it in the admin and content panel.
I think this was maybe added for testing and forgotten :D